README.md

```diff
@@ -36,7 +36,7 @@
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
```

Comment:
- [x] Hello Divine, Well done removing this instruction line from your readme. Kindly, remove the other instructions from your readme file so that it looks professional and well-documented.